### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by krishvoor**
This PR Aims at providing fix for #6 via updating the relevant Documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ Follow these steps to kickstart OCP installation on KVM
 On your Terraform client machine:
 1. `git clone https://github.com/ocp-power-automation/ocp4-upi-kvm.git`
 2. `cd ocp4_upi_kvm`
+3. `go get -u github.com/dmacvicar/terraform-provider-libvirt`
+
+   `cp $GOPATH/bin/terraform-provider-libvirt ~/.terraform.d/plugins/darwin_amd64/`
```
Comment (yussufsh) [inline README.md:50]: Here `darwin_amd64` could be `linux_amd64` for linux OS.
Better to copy the plugin binary to repo dir.

**Comment by yussufsh** [comment]
I am not in favor of mentioning the steps required for external dependcies in the README here. This part is already documented on the referenced link and we have mentioned it in this repo as well.

Most we could do is move the reference up to Pre-requisite section without giving detailed steps. (Which could in turn depend on other things like machine arch, OS and even Go version).

What you have to say @bpradipt ?

File: README.md (hunk 1 above)
Comment (krishvoor) [reply README.md:50]: Please confirm if this looks good -
`cp $GOPATH/bin/terraform-provider-libvirt ocp-power-automation` instead ?

Also, as suggested, I will move this particular piece of documentation to Pre-req Section.

**Comment by bpradipt** [comment]
> I am not in favor of mentioning the steps required for external dependcies in the README here. This part is already documented on the referenced link and we have mentioned it in this repo as well.
> 
> Most we could do is move the reference up to Pre-requisite section without giving detailed steps. (Which could in turn depend on other things like machine arch, OS and even Go version).
> 
> What you have to say @bpradipt ?

Yeah I tend to agree...  I think what we can do is create a separate doc to describe libvirt terraform provider build and install since this is not part of default TF providers and link it in the main readme. 

@krishvoor can you do that ?

**Comment by krishvoor** [comment]
> > I am not in favor of mentioning the steps required for external dependcies in the README here. This part is already documented on the referenced link and we have mentioned it in this repo as well.
> > Most we could do is move the reference up to Pre-requisite section without giving detailed steps. (Which could in turn depend on other things like machine arch, OS and even Go version).
> > What you have to say @bpradipt ?
> 
> Yeah I tend to agree... I think what we can do is create a separate doc to describe libvirt terraform provider build and install since this is not part of default TF providers and link it in the main readme.
> 
> @krishvoor can you do that ?

Okay I can do that.. will create a doc .. capture the steps required and link them back to README.md file..

**Comment by krishvoor** [comment]
@bpradipt Updated documentation relevantly - new PR is via https://github.com/ocp-power-automation/ocp4-upi-kvm/pull/22

**Pull request closed without merging**